### PR TITLE
Added galera recovery option

### PR DESCRIPTION
**Pull request by ovk6**

### CODE REVIEW

**Comment by aav66** [comment]
LGTM

File: 10/10.2/docker-entrypoint.sh [hunk 1]
```diff
@@ -91,6 +98,25 @@ fi
 if [ "$1" = 'mysqld' -a -z "$wantHelp" ]; then
 	# still need to check config, container may have started with --user
 	_check_config "$@"
+
+	# run Galera auto-recovery
```
Comment (wgrzelak) [inline 10/10.2/docker-entrypoint.sh]: Can you think about a test for this?

Maybe some simple which will help is avoid deleting GALERA_RECOVERY in the future.

File: 10/10.1/Dockerfile [hunk 2]
```diff
@@ -170,7 +170,7 @@ RUN set -ex; \
 
 VOLUME /var/lib/mysql
 
-COPY docker-entrypoint.sh /usr/local/bin/
+COPY --chown=mysql docker-entrypoint.sh /usr/local/bin/
```
[the remark this answers is not in the document]
Comment (ovk6) [reply 10/10.1/Dockerfile]: Because this entrypoint is run with mysql user and it fails otherwise.

File: 10/10.2/docker-entrypoint.sh (hunk 1 above)
Comment (ovk6) [reply 10/10.2/docker-entrypoint.sh]: I will prepare some tests in other PR.

**Pull request merged**